README.md

```diff
@@ -83,6 +83,25 @@ The project is structured as follows:
 - `/.storybook/` - Contains the configuration for storybook
 - `/scripts/` - Contains scripts that are used for development and deployment
 
+## Recommended Workflow for creating new Tasks (WIP)
```

Comment:
Adding a checklist of steps for creating a new Task; probably still missing some steps so feel free to add if you see anything missing